src_kt/java/arcs/util/Base64.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.util
+
+/** Implementations of Base-64 encoding/decoding. */
+object Base64 {
```

Comment:
I found this guide https://kotlinlang.org/docs/tutorials/mpp/multiplatform-library.html which has a small base64 implementation in kotlin native, and says it's published to maven. Could we just use that instead?

Reply:
If it's published as a klib (multiplatform), then it would work, at least, in the open source. I'm sure of the state of multiplatform support internally. I hope it works. :)